pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/TemplateDao.sql.stg

```diff
@@ -1,27 +1,25 @@
 group TemplateDao;
 
 queryAllTranslatedVariablesByTemplateIdsAndLangId() ::= <<
-select
-  tv.template_id, tv.variable_name, coalesce(i18n_ts.substitution_value, i18n_ts_def.substitution_value) as substitution_value
-from
-  template_variable as tv
-  left join i18n_template_substitution as i18n_ts on
-    i18n_ts.template_variable_id = tv.template_variable_id and
-    i18n_ts.language_code_id = :langCodeId
-  left join revision as i18n_ts_rev on
-    i18n_ts_rev.revision_id = i18n_ts.revision_id
-  left join i18n_template_substitution as i18n_ts_def on
-    i18n_ts_def.template_variable_id = tv.template_variable_id and
-    i18n_ts_def.language_code_id = :defaultLangCodeId
-  left join revision as i18n_ts_def_rev on
-    i18n_ts_def_rev.revision_id = i18n_ts_def.revision_id
-where
-  tv.template_id in (\<templateIds\>) and
-  ( (i18n_ts_rev.start_date \<= :timestamp and (i18n_ts_rev.end_date is null or :timestamp \< i18n_ts_rev.end_date))
-    or
-    (i18n_ts_def_rev.start_date \<= :timestamp and (i18n_ts_def_rev.end_date is null or :timestamp \< i18n_ts_def_rev.end_date))
-  ) and
-  (i18n_ts.substitution_value is not null or i18n_ts_def.substitution_value is not null)
+select tv.template_id, tv.variable_name,
```

Comment:
I had to change up this query because I was having issues with it where it will return the wrong version of the text. Given the way we were doing joins in the old query, it was creating a "cross-product" of sorts between the translation text and the "default" translation text. And when we add revisioned translations into the mix, it was returning the latest revisions. The timestamp time-range comparison was correct, but because of the joins, it also allowed newer revisions to pass through the `where` criteria. Therefore, I changed it to use subqueries and `coalesce()` in order to keep the same semantics but filter out un-matched revisions.